day_7/10_speckled_frogs.js

```diff
@@ -0,0 +1,9 @@
+for (var i=1; i<2; i++){
```

Comment:
One way to make this a little cleaner is to store the common text as variables, and just reference the variable names in the for loop.  For example you could store the string `speckled frogs sat on a log \n\ eating some most delicious bugs. \n\ One jumped in a pool where it was nice and cool` in a variable.